client/src/components/payment/payment-history-modal.vue

```diff
@@ -1,27 +1,29 @@
 <template>
-  <v-dialog max-width="40%" :value="isVisible" persistent v-if="payments">
-    <v-card class="pa-9">
-      <v-card-text class="h3">Payment History</v-card-text>
-      <v-card-text class="copy-normal">
+  <v-dialog width="30rem" :value="isVisible" persistent v-if="payments">
+    <v-card>
+
+      <v-card-title>Payment History</v-card-title>
+
```

Comment:
Super minor, is there extra spacing here?

Reply:
No, I wanted each block to be separate.

Reply:
K i take that back, i prefer the spacing in the template as well. Just not the case in many of the templates but would be cool to progress towards that. So ignore the first comment :)